modular_skyrat/modules/modular_implants/code/nifsofts/hivemind.dm

```diff
@@ -170,10 +170,13 @@ GLOBAL_LIST_EMPTY(hivemind_users)
 	var/linking_protection = TRUE
 
 /datum/component/mind_linker/nif
+	speech_action = FALSE
+	linking_protection = FALSE
+
 	///What is the name of the hivemind? This is mostly here for the TGUI selection menus.
```

Comment:
you didn't move the code doc comment and kept 177-178 despite being a dupe of 173-174

Reply:
ugh, my VSCode Github setup was having issues.